api/core/Directus/Database/TableGateway/RelationalTableGateway.php

```diff
@@ -67,10 +67,12 @@ class RelationalTableGateway extends BaseTableGateway
         'nbetween' => ['operator' => 'between', 'not' => true],
     ];
 
-    public function manageRecordUpdate($tableName, $recordData, $activityEntryMode = self::ACTIVITY_ENTRY_MODE_PARENT, &$childLogEntries = null, &$parentCollectionRelationshipsChanged = false, $parentData = [])
+    public function manageRecordUpdate($recordData, $tableName = null, $activityEntryMode = self::ACTIVITY_ENTRY_MODE_PARENT, &$childLogEntries = null, &$parentCollectionRelationshipsChanged = false, $parentData = [])
```

Comment:
This is a risky step, this may break current user code, it would've been better to write a new method and mark this as deprecated.